internal/mailer/mailer.go

```diff
@@ -29,8 +29,12 @@ type Mailer interface {
 func NewMailer(globalConfig *conf.GlobalConfiguration) Mailer {
 	mail := gomail.NewMessage()
 
-	// so that messages are not grouped under each other
-	mail.SetHeader("Message-ID", fmt.Sprintf("<%s@gotrue-mailer>", uuid.Must(uuid.NewV4()).String()))
+	mail.SetHeaders(map[string][]string{
+		// Make the emails explicitly set to be HTML formatted (to cover older email clients)
+		"Content-Type": {"text/html; charset=utf-8"},
```

Comment:
Thanks for the contribution! There's an existing `SetBody` within the mailme package- https://github.com/supabase/mailme/blob/main/mailme.go#L71

Not sure if that will override the header. set here - if it doesn't then I think this should be good to go!

Reply:
It sets the body type there, but it doesn't make it clear in the header which confuses some old email clients. 
They are different/ won't override one another.

Reply:
Cool, thanks! for the explanation